homeassistant/components/sensor/tautulli.py

```diff
@@ -0,0 +1,133 @@
+                          host, port, monitored_users, schema)])
+
+
+class Tautulli(Entity):
+    """Representation of a Sensor."""
+
+    def __init__(self, api_key, monitored_variables,
+                 host, port, users, schema):
+        """Initialize the sensor."""
+        import pytautulli
+        self.tautulli = pytautulli
+        self._state = STATE_UNAVAILABLE
+        self._api_key = api_key
+        self._monitored_variables = monitored_variables
+        self._host = host
+        self._port = port
+        self._user = users
+        self._schema = schema
+        self._data = {}
+        self.update()
+
+    def update(self):
+        """Update sensor value."""
+        most_stats = self.tautulli.get_most_stats(self._host,
+                                                  self._port,
+                                                  self._api_key,
+                                                  self._schema)
+        for key in most_stats:
+            self._data[str(key)] = str(most_stats[key])
+
+        sever_stats = self.tautulli.get_server_stats(self._host,
+                                                     self._port,
+                                                     self._api_key,
+                                                     self._schema)
+        for key in sever_stats:
+            self._data[str(key)] = str(sever_stats[key])
+
+        users = self.tautulli.get_users(self._host,
+                                        self._port,
+                                        self._api_key,
+                                        self._schema)
+        for user in users:
+            if user != 'Local' and (user in self._user or self._user is None):
+                userstate = self.tautulli.get_user_state(self._host,
+                                                         self._port,
+                                                         self._api_key,
+                                                         user,
+                                                         self._schema)
+                self._data[str(user)] = {}
+                self._data[str(user)]['activity'] = str(userstate)
+                attrlist = self.tautulli.get_user_activity(self._host,
+                                                           self._port,
+                                                           self._api_key,
+                                                           user,
+                                                           self._schema)
+                for key in self._monitored_variables:
+                    try:
+                        self._data[str(user)][str(key)] = str(attrlist[key])
+                    except KeyError:
+                        self._data[str(user)][str(key)] = ""
+        self._state = sever_stats['count']
+
+    @property
+    def name(self):
+        """Return the name of the sensor."""
+        return 'Tautulli'
```

Comment:
Add `CONF_NAME` with the default `Tautulli` to the configuration. This allows the user to overwrite the name of the sensor without a detour over `customize:`